pam_login_without_env.py

```diff
@@ -0,0 +1,13 @@
+import irods.session
+#s = h.make_session()
```

Comment:
Remove if not needed.

Reply:
Yes, and ... again ... I think this is yet another thing that will go away.  Apologies for its inclusion.

Reply:
File removed.